src/components/Refresh/Refresh.tsx

```diff
@@ -0,0 +1,93 @@
+import * as React from 'react';
+import { Button, MenuItem, Icon, DropdownButton } from 'patternfly-react';
+
+import { config } from '../../config';
+import { PollIntervalInMs } from '../../types/Common';
+
+type Props = {
+  id: string;
+  pollInterval?: PollIntervalInMs;
+  handleRefresh: () => void;
+  onSelect: (selected: PollIntervalInMs) => void;
+};
+
+type State = {
+  pollInterval?: PollIntervalInMs;
+  pollerRef?: number;
+};
+
+class Refresh extends React.Component<Props, State> {
+  static readonly POLL_INTERVALS = config().toolbar.pollInterval;
```

Comment:
Just my curiosity, is better to use "static readonly" vs "const" on these use cases ?